staging/copy.sh

```diff
@@ -202,6 +207,9 @@ find "${CLIENT_REPO_TEMP}" -type f \( \
     -name "*.sh" \
     \) -delete
 
+echo "remove cyclical godep"
```

Comment:
> this is a little scary.

Yeah, but there's an entire separate go program devoted to managing the file, so I'm not the guy who discover weird godep behavior.  In all honesty, this whole, "copy, mutate, revendor, tweak vendor" which pre-exists my pull is pretty wild.